package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless",
-  "version": "1.43.0",
+  "version": "1.44.0",
```

Comment:
I just merged some other PRs in the meantime, so it might be worthwhile to rebase this PR.

Other than that we could also include the `CHANGELOG.md` changes later on here.

Reply:
yup. i'll run that next week when we release on tuesday.

Reply:
(the change log, the branch is uptodate with master :) )